src/main/java/com/thealgorithms/maths/SecondMinMax.java

```diff
@@ -0,0 +1,46 @@
+package com.thealgorithms.maths;
+
+public class SecondMinMax {
```

Comment:
You can make a _proper utility class_ of this.

```suggestion
public final class SecondMinMax {
    private SecondMinMax() {
    }
```

Reply:
Sure will change this 👍